languages.toml

```diff
@@ -3840,11 +3840,14 @@ source = { git = "https://github.com/Decurity/tree-sitter-circom", rev = "021505
 name = "snakemake"
 scope = "source.snakemake"
 roots = ["Snakefile", "config.yaml", "environment.yaml", "workflow/"]
-file-types = ["smk", "Snakefile"]
+file-types = ["smk", { glob = "Snakefile" } ]
 comment-tokens = ["#", "##"]
 indent = { tab-width = 2, unit = "  " }
 language-servers = ["pylsp" ]
 
+[language.formatter]
+command = "snakefmt"
+args = ["-"]
```

Comment:
This was not so clearly documented, but in the warning callout at the start of the Github [Readme](https://github.com/snakemake/snakefmt?tab=readme-ov-file#snakefmt), they state:

> You can also pipe the file in from stdin, which will print it to the screen.

Later, in the last example of [Basic Usage](https://github.com/snakemake/snakefmt?tab=readme-ov-file#basic-usage) they show how to write the output to stdout with
```Snakemake
snakefmt - < Snakefile
```
It turns out, that by removing the `< Snakefile` part above, `snakefmt` takes input from stdin and sends the output to stdout.